app_legacy/Helpers/database/player-achievement.php

```diff
@@ -179,47 +181,6 @@ function insertAchievementUnlockIntoAwardedTable(string $user, int $achIDToAward
     return $dbResult !== false;
 }
 
-function getUsersRecentAwardedForGames(string $user, string $gameIDsCSV, int $numAchievements): array
```

Comment:
this code has been merged into `getUserProgress`, as they were called together by `getUserPageInfo` and fetched similar data.